src/machines/workflow_machines.rs

```diff
@@ -442,11 +443,15 @@ impl WorkflowMachines {
 
     /// Iterate the state machines, which consists of grabbing any pending outgoing commands from
     /// the workflow, handling them, and preparing them to be sent off to the server.
```

Comment:
We need to update the docstring to talk about the return value

Reply:
Done